src/scss/main.scss

```
@@ -2,22 +2,11 @@
 @import "~bootstrap/scss/bootstrap";
 
 @import "variables";
+@import "mixin";
```

Better use one naming pattern, add `s`, please.
```suggestion
@import "mixins";
```